tensorflow/python/estimator/canned/dnn.py

```diff
@@ -59,7 +81,9 @@ def _dnn_logit_fn_builder(units, hidden_units, feature_columns, activation_fn,
     dropout: When not `None`, the probability we will drop out a given
       coordinate.
     input_layer_partitioner: Partitioner for input layer.
-    batch_norm: Whether to use batch normalization after each hidden layer.
+    batch_norm: A boolean indicating whether to to use batch normalization
+      after each hidden layer with default options, or a `_BatchNormOptions`
```

Comment:
here instead of referring a private symbol '_BatchNormOptions', mention experimental_batch_norm_options